src/main/java/ProductPage.java

```diff
@@ -4,15 +4,16 @@
 public class ProductPage {
 
     WebDriver driver;
-    public static final String idTextDispayed = "productTitle";
+    By TextDispayedSelector = By.id("productTitle");
 
     public ProductPage(WebDriver driver){
         this.driver = driver;
     }
 
-    public boolean isNintendoSwitchContained(String expectedText){
-        String displayedText = driver.findElement(By.id(idTextDispayed)).getText();
+    public String isNintendoSwitchContained(){
```

Comment:
the name of the function does not correspond with the action done. This function should be named something like `getProductTitle()`